packaging/linux/build_and_push_packages.sh

```diff
@@ -90,14 +90,12 @@ release_prerelease() {
 
   json_tmp=`mktemp`
   echo "Writing version into JSON to $json_tmp"
-  cat > "$json_tmp" <<END
-{
-"version": "$version",
-"name": "v$version",
-"description": "Latest Linux release",
-"type": 0
-}
-END
+
+  echo "Loading release tool"
+  "$client_dir/packaging/goinstall.sh" "github.com/keybase/release"
+  release_bin="$GOPATH/bin/release"
```

Comment:
Oh crap I missed a bug. In the Docker build, GOPATH isn't set here, and this step fails. We set it below at line 103. Didn't show up in my testing because of course I do have GOPATH set on my own machine :p